test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -494,8 +502,14 @@ void codecFuzz(const test::common::http::CodecImplFuzzTestCase& input, HttpVersi
         headers_with_underscores_action);
   }
 
-  ReorderBuffer client_write_buf{*server};
-  ReorderBuffer server_write_buf{*client};
+  // We track whether the connection should be closed for HTTP/1, since stream resets imply
+  // connection closes.
+  bool should_close_connection = false;
+
+  // The buffers will be blocked from dispatching data if should_close_connection is set to true.
+  // This prevents sending data if a stream reset occurs during the test cleanup when using HTTP/1.
+  ReorderBuffer client_write_buf{*server, should_close_connection};
+  ReorderBuffer server_write_buf{*client, should_close_connection};
```

Comment:
shouldn't the server buffer hold a reference to an always false?

Reply:
After digging through the code again I'm not entirely certain that this can only happen on the client buffer.
Let me know if you think otherwise, and I'll happily update the code.

Reply:
Whoops, yes, all actions and data dispatching should stop. thanks!